input/pagecontent/2-5-EventNotification.md

```diff
@@ -90,49 +90,49 @@ Field    | Optionality | Type     | Description
+  <figcaption><b>Figure: Event Notification Error Sequence</b></figcaption>
 </figure>
 
 More information on the source of notification errors and how to resolve them can be found in [Synchronization Considerations](4-2-syncconsiderations.html).
 
 ### Hub Generated `syncerror` Events
 
-In addition to distributing [`syncerror`](3-2-1-syncerror.html) events sent by one application to other subscribed applications, the Hub MAY generate and communicate [`syncerror`](3-2-1-syncerror.html) events to applications under the following conditions -- 
+In addition to distributing [`syncerror`](3-2-1-syncerror.html) events sent by one Subscriber to other Subscribers, the Hub MAY generate and communicate [`syncerror`](3-2-1-syncerror.html) events to Subscribers under the following conditions: 
 
-1. A subscribed application's WebSocket connection is closed with any Connection Close Reason other than 1000 (normal closure) or 1001 (going away) (see [WebSocket RFC](https://www.rfc-editor.org/rfc/rfc6455.html#section-7.1.6) and [WebSocket Status Codes](https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4))
+1. A Subscriber's WebSocket connection is closed with any Connection Close Reason other than 1000 (normal closure) or 1001 (going away) (see [WebSocket RFC](https://www.rfc-editor.org/rfc/rfc6455.html#section-7.1.6) and [WebSocket Status Codes](https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4))
 
-2. And, the subscribed application, does not respond to a FHIRcast event within 10 seconds or an order of magnitude lower than the subscription time-out.
+2. A Subscriber does not respond to a FHIRcast event within 10 seconds or an order of magnitude lower than the subscription time-out.
 
 > Implementer input is solicited on the amount and specificity of time, in the above.
 
-[`syncerror`](3-2-1-syncerror.html) events are distributed only to applications which have subscribed to `syncerror`s.
+[`syncerror`](3-2-1-syncerror.html) events are distributed only to Subscribers which have subscribed to `syncerror` events.
 
-Upon communicating a `syncerror` resulting from an unresponsive app, the Hub SHALL unsubscribe the application.
+Upon communicating a `syncerror` resulting from an unresponsive Subscriber, the Hub SHALL unsubscribe the Subscriber.
 
 The Hub SHALL NOT generate [`syncerror`](3-2-1-syncerror.html) events in the following situations:
 
-1. If a client fails to respond to a [`heartbeat`](3-2-2-heartbeat.html) event (because occasional missed heartbearts are expected and are not a context synchronization failure).
-2. The application closes its WebSocket connection to the Hub with a [Connection Close Reason](https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4.1) of 1000 (normal closure) or 1001 (going away).  
+1. A Subscriber fails to respond to a [`heartbeat`](3-2-2-heartbeat.html) event (because occasional missed heartbearts are expected and are not a context synchronization failure).

```

Comment:
Additional typo: because occasional missed *heartbearts*, should be heartbeats.

Reply:
Corrected